src/Compilers/VisualBasic/Portable/Analysis/FlowAnalysis/VisualBasicDataFlowAnalysis.vb

```diff
@@ -202,6 +210,26 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             End Get
         End Property
 
+        Public Overrides ReadOnly Property CapturedInside As ImmutableArray(Of ISymbol)
+            Get
+                If Me._capturedInside.IsDefault Then
+                    AnalyzeReadWrite()
+                End If
+
+                Return Me._capturedInside
+            End Get
+        End Property
+
+        Public Overrides ReadOnly Property CapturedOutside As ImmutableArray(Of ISymbol)
+            Get
+                If Me._capturedOutside.IsDefault Then
+                    AnalyzeReadWrite()
+                End If
+
+                Return Me._capturedOutside
+            End Get
+        End Property
```

Comment:
Not related to this PR, but it seems that we could largely factor the C# and VB code.

Reply:
Filed https://github.com/dotnet/roslyn/issues/23786 to follow-up on this.